semgrep-core/src/pfff/lang_ml/parsing/ast_ml.ml

```diff
@@ -303,7 +303,7 @@ and type_def_kind =
   (* or type *)
   | AlgebraicType of constructor_decl list
   (* and type *)
-  | RecordType   of field_decl list bracket
+  | RecordType of field_decl list bracket
```

Comment:
during vacation I'll run ocamlformat on all the pfff code.
Right now it's excluded in pre-commit.